canonical-web-and-design/maas-ui
# feat(ui): add alias rows to the network table

## Done

- Add the rows that display the links/aliases in the network tab of machine details.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the network tab in legacy for a machine that can edit network (New, Ready, Allocated or Broken.)
- Create a Bond (if one doesn't exist already).
- use the action menu to add an alias.
- open the network tab for the same machine in the react client
- you should see the alias row and it should have the same data as the legacy page.

## Fixes

Fixes: #1974.
